client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,223 @@
+# Guided Tours API
+
+Guided Tours are declared in JSX as a tree of elements. All of them are available as named exports from `client/layout/guided-tours/config-elements`.
+
+## All-in example
```

Comment:
instead of having the Step here, move it to ## Step below. 

here you might want to add some more general advice, like, a Tour is a series of Step elements. the tour gets triggered either if selectors match or when it's forced to start through a query argument. (document the query arg somewhere, maybe where you document the tour name prop) the user can then step through the tour and get Calypso aspects explained. can be interactive in that Step element can wait for a user to do certain actions such as clicking a menu item. we can also skip tour steps depending on selectors, e.g. skip a single Step if that one doesn't work well on mobile.